### DISCUSSION TITLE
"Umber" is a shade of what color?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#371] [Galaxy Trivia] [answered]
**Hint:** 🍫

**Comment by thuongtruong1009** [comment, marked as answer]
"Umber" is a shade of brown. It is a natural earth pigment that is typically a dark, reddish-brown color. The name "umber" comes from the Italian word "terra d'ombra," which means "earth of shadow." The pigment is made from iron oxide and manganese oxide and has been used as a coloring agent for centuries. It is commonly used in art and painting to create warm, natural-looking browns, and it is also used in industries such as ceramics and textiles to dye fabrics and glazes.
